app/src/main/java/com/example/encoratask/ui/view/CharacterDetailActivity.kt

```diff
@@ -1,28 +1,29 @@
-package com.example.encoratask
+package com.example.encoratask.ui.view
 
 import android.content.Intent
 import android.os.Bundle
+import android.view.MenuItem
+import androidx.appcompat.app.AppCompatActivity
+import com.example.encoratask.R
 import com.google.android.material.floatingactionbutton.FloatingActionButton
 import com.google.android.material.snackbar.Snackbar
-import androidx.appcompat.app.AppCompatActivity
-import android.view.MenuItem
 
 /**
  * An activity representing a single Item detail screen. This
  * activity is only used on narrow width devices. On tablet-size devices,
  * item details are presented side-by-side with a list of items
- * in a [ItemListActivity].
+ * in a [CharacterListActivity].
  */
-class ItemDetailActivity : AppCompatActivity() {
+class CharacterDetailActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_item_detail)
+        setContentView(R.layout.activity_character_detail)
         setSupportActionBar(findViewById(R.id.detail_toolbar))
 
         findViewById<FloatingActionButton>(R.id.fab).setOnClickListener { view ->
```

Comment:
This piece of code will fail since you already removed the floating button in the xml

Reply:
This is for the CharacterDetailActivity, the floating button I removed was on the CharacterListActivity so it does not fail here.